src/bot/commands/utilities/poll.ts

```diff
@@ -0,0 +1,59 @@
+import {Command, Input} from '@api';
+import {TextChannel} from 'discord.js';
+import {UserPoll} from "@libraries/utilities/create_poll";
+import {Emoji} from "@libraries/emoji";
+
+export class Poll extends Command {
+    constructor() {
+        super({
+            name: 'poll',
+            description: 'Lets members vote on between 2-6 options',
+            arguments: [
+                {
+                    name: 'question',
```

Comment:
```suggestion
                    name: 'question',
                    description: 'The question to start a poll for, in quotation marks.',
```

Help information looks gross without argument descriptions.

Reply:
Of course. I forgot the framework handled that for multiple parameters.